recipes/liboptv/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: optv
+  version: 0.2.3
+
+source:
+  url: https://github.com/OpenPTV/openptv/archive/0.2.3.tar.gz
+  sha256: 21b8709dedc8696ff454d83d2d6af582252dbd9cf621f5d78ecdd64ce0af48cb
+
+build:
+  number: 0
+  script: cd py_bind && python setup.py prepare && python setup.py install
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - numpy
```

Comment:
You need to also specify the numpy version which you need
In you project is hardcoded the version ``1.16.1``. So you should replace this line by
```
- numpy 1.16.1
```